charts/operator-wandb/charts/flat-runs-field-updater/templates/hpa.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.autoscaling.enabled }}
+---
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "flat-runs-field-updater.fullname" . }}
+  labels:
+    {{- include "flat-runs-field-updater.labels" . | nindent 4 }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "flat-runs-field-updater.fullname" . }}
+  minReplicas: {{ .Values.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.autoscaling.maxReplicas }}
+  metrics:
+    {{- if .Values.autoscaling.targetCPUUtilizationPercentage }}
+    - type: Resource
+      resource:
+        name: cpu
+        target:
+          type: Utilization
+          averageUtilization: {{ .Values.autoscaling.targetCPUUtilizationPercentage }}
+    {{- end }}
+    {{- if .Values.autoscaling.targetMemoryUtilizationPercentage }}
+    - type: Resource
+      resource:
+        name: memory
+        target:
+          type: Utilization
+          averageUtilization: {{ .Values.autoscaling.targetMemoryUtilizationPercentage }}
+    {{- end }}
+{{- end }}
```

Comment:
Could we use the naming schema and patterns found in [bitnami](https://github.com/bitnami/charts/blob/5e4910e8bc2b7624e34f91950cd378bca420000b/bitnami/nginx-ingress-controller/templates/controller-hpa.yaml#L8)